modules/RTC/TraceablePeerConnection.js

```diff
@@ -887,10 +888,8 @@ TraceablePeerConnection.prototype.generateNewStreamSSRCInfo = function () {
         for (let i = 0; i < SIMULCAST_LAYERS; i++) {
             ssrcInfo.ssrcs.push(SDPUtil.generateSsrc());
         }
-        ssrcInfo.groups.push({
-            primarySSRC: ssrcInfo.ssrcs[0],
-            group: {ssrcs: ssrcInfo.ssrcs.join(" "), semantics: "SIM"}});
-        ssrcInfo;
+        ssrcInfo.groups.push(
+            { ssrcs: ssrcInfo.ssrcs.slice(), semantics: "SIM" });
```

Comment:
personally i think it's easier to read if the fields of an object are split across new lines (similar to the way it was before), but maybe just a personal preference